guides/doc-Release_Notes/topics/katello-4.8.1.adoc

```diff
@@ -0,0 +1,24 @@
+= Katello 4.8.1
+
+A full list of changes is available on https://projects.theforeman.org/issues?set_filter=1&sort=id%3Adesc&status_id=closed&f%5B%5D=cf_12&op%5Bcf_12%5D=%3D&v%5Bcf_12%5D%5B%5D=1688[Redmine]
+
+== Katello
+
+* DB Migration fails because of missing Setting::Content - https://projects.theforeman.org/issues/36343[#36343]
+* Enable tracer on host page (debian os) is always on - https://projects.theforeman.org/issues/36297[#36297]
+
+=== Content Views
+
+* Getting "NoMethodError undefined method \`get_status\' for nil:NilClass" when publishing content view - https://projects.theforeman.org/issues/36303[#36303]
```

Comment:
I think this doesn't need escaping because it doesn't render well. I never understand escaping in asciidoc.
```suggestion
* Getting "NoMethodError undefined method `get_status' for nil:NilClass" when publishing content view - https://projects.theforeman.org/issues/36303[#36303]
```

Looks like it's repeated a few times.